File: Server/src/main/java/com/example/journal/model/Story.java
package com.example.journal.model;


import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.*;

import javax.persistence.*;
import java.time.LocalDate;
import java.time.LocalDateTime;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
@Table(name = "story")
public class Story {

    @Id
    private Long id;

    @Column(name = "title")
    private String title;

    @Column(name = "date")
    private String date;

    @Column(name = "emotion")
    private String emotion;

    @Column(name = "motivational_message")
    private String motivationalMessage;

    @Column(name = "text")
    private String text;
}

File: app/src/main/java/com/example/journal/model/Story.kt
package com.example.journal.model

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Story(
    val id: Int,
    var title: String, var text: String, var date: org.threeten.bp.LocalDate,
    var emotion: String, var motivationalMessage: String): Parcelable {
}